src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/AnnexTests.kt

```diff
@@ -0,0 +1,22 @@
+package org.vaccineimpact.api.databaseTests.tests
```

Comment:
Am I right in thinking if you remove this class for now (include in the second PR instead) the code will compile? Would be good not to have to merge in code with non passing tests!